core/src/Modal/Modal.less

```diff
@@ -27,11 +29,7 @@
 .Modal {
   background-color: @zesty-white;
   border-radius: 4px;
-  display: flex;
-  flex-direction: column;
-  max-width: 50%;
-  margin: 5% auto 0;
-  min-width: 462px;
```

Comment:
We do not want to remove the min-width. After a certain width we have to begin to question why a modal is being used and how useful is it.

Reply:
restored original styles